ParkenDD/AppDelegate.swift

```diff
@@ -27,7 +27,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
 	func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
         Location.manager.requestWhenInUseAuthorization()
-
+		Location.manager.startUpdatingLocation()
```

Comment:
Indentation issues again.

Reply:
My Xcode is set to having a prefer indent with Tabs. Tab width is set to 4 spaces and Indent width is set also to 4 spaces. I also make sure to Command + i . I will check this...